docs/docs/cmd/spo/group/group-user-add.md

```diff
@@ -0,0 +1,36 @@
+# spo group user add
+
+Add a user or multiple users to SharePoint Group
+
+## Usage
+
+```sh
+m365 spo group user add [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the SharePoint group is available
+
+`-i, --groupId <groupId>`
```

Comment:
I didn't catch it in the spec, but let's remove the short `-i` here. It would be ambiguous if we ever extended the command with support for specifying `userId`.

Reply:
Sure. That make sense considering the extension of the command.
Have updated the specs as well so that [specs ](https://github.com/pnp/cli-microsoft365/issues/2483#issue-916289528)are consistent.